exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/SystemTableScan.java

```diff
@@ -85,6 +83,12 @@ public PhysicalOperator getNewWithChildren(List<PhysicalOperator> children) {
   public void applyAssignments(List<DrillbitEndpoint> endpoints) {
   }
 
+  @JsonIgnore
+  @Override
+  public List<SchemaPath> getColumns() {
+    return super.getColumns();
+  }
```

Comment:
See note above. Columns are part of the logical plan and must be serialized. To see this, do an `EXPLAIN PLAN FOR`. You'll see that the JSON includes the column projection for other group scans.

Reply:
Added columns to the plan.